torch/_torch_docs.py

```diff
@@ -2728,20 +2730,26 @@ def merge_dicts(*dicts):
     tensor([-1.,  1., -1., -1.])
 """.format(**common_args))
 
-add_docstr(torch.floor_divide,
-           r"""
+add_docstr(torch.floor_divide, r"""
 floor_divide(input, other, *, out=None) -> Tensor
 
-Return the division of the inputs rounded down to the nearest integer. See :func:`torch.div`
-for type promotion and broadcasting rules.
+Computes the floor of :attr:`input` divided by :attr:`other`.
 
 .. math::
     \text{{out}}_i = \left\lfloor \frac{{\text{{input}}_i}}{{\text{{other}}_i}} \right\rfloor
 
 """ + r"""
+
+Supports :ref:`broadcasting to a common shape <broadcasting-semantics>`,
+:ref:`type promotion <type-promotion-doc>`, and integer and float inputs.
+
+.. note:: Floor division in PyTorch is like floor division in Python
```

Comment:
nit: torch.floor_divide performs floor division as in Python and NumPy.  This differs from integer division in C++, which truncates instead of flooring the quotient.  E.g. in Python, -5 // 2 == -3, but in C++ -5 // 2 = -2.

Reply:
perhaps its better to refer to this as C-style, assuming C and C++ behave the same.

Reply:
SG.